compiler/ras/LimitFile.cpp

```diff
@@ -1326,8 +1326,11 @@ TR_Debug::methodSigCanBeCompiledOrRelocated(const char *methodSig, TR_FilterBST
       {
       if (compOrReloFilter->excludedMethodFilter)
          {
-         // Excluded methods SHOULD be compiled in this case.
-         //
+         // -Xjit:ifExcluded() option is used to set alternate compile options for the excluded methods.
```

Comment:
I propose rewording slightly:
```
// The -Xjit:ifExcluded(...) option set is used to set alternate compile options for methods
// that are excluded from compilation.  This can be useful for debugging timing-sensitive 
// optimization bugs where dropping optimization levels can make problems go away.
//
// If there is an excludedMethodFilter set then excluded methods should be compiled
// using that option set.
//
```

Reply:
Thanks @0xdaryl . I updated the pull request with single commit with your suggested update to the comment and commit title